content/influxdb3/core/reference/processing-engine.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Processing engine reference
+description: >
+  Use the InfluxDB 3 Processing engine to run Python code directly in your InfluxDB 3 Core database to automatically process data and respond to database events.
```

Comment:
```suggestion
  The InfluxDB 3 Processing engine is an embedded Python virtual machine that runs inside the InfluxDB database. It executes Python code in response to database events without requiring external application servers or middleware.
```

We typically (but not strictly) write references in third person, active voice, with the feature, command, etc as the subject ("The Processing engine runs..."). Ideally, (and in this case), you can use use the intro paragraph as the description.

Generally, reference topics are more about the details and how the thing works rather than how to use it. It doesn't always play well, but that's what we aim for.